modulemd/v2/tests/test-modulemd-merger.c

```diff
@@ -100,6 +104,47 @@ merger_test_deduplicate (CommonMmdTestFixture *fixture,
   g_assert_cmpstr (baseline, ==, deduplicated);
 }
 
+static void
+merger_test_merger_with_real_world_data (CommonMmdTestFixture *fixture,
+                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleIndex) fedora_index = NULL;
+  g_autoptr (ModulemdModuleIndex) updates_index = NULL;
+  g_autoptr (ModulemdModuleIndex) merged_index = NULL;
+  g_autoptr (ModulemdModuleIndexMerger) merger = NULL;
+  g_autofree gchar *yaml_path = NULL;
+  g_autoptr (GPtrArray) failures = NULL;
+  g_autoptr (GError) error = NULL;
+  
+  yaml_path =
```

Comment:
You allocate this, then don't free it anywhere. It gets masked by the other allocation at line 126, resulting in leaked memory.

Reply:
I have removed the line 126 and changed  the first yaml_path to f29-updates.yaml file to make it non identical with the deduplicated test as suggested by you.